### PR TITLE
Copy env and procfile

### PR DESCRIPTION
**Pull request by weargoggles**

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+set -e
+
+export_env_dir() {
+  env_dir=$3
+  blacklist_regex=${3:-'^(PATH|GIT_DIR|CPATH|CPPATH|LD_PRELOAD|LIBRARY_PATH)$'}
+  if [ -d "$env_dir" ]; then
+    for e in $(ls $env_dir); do
+      echo "$e" | grep -qvE "$blacklist_regex" &&
+      export "$e=$(cat $env_dir/$e)"
+      :
```
Comment (will-ockmore) [inline bin/compile]: Noob bash comment - why do you need a colon operator here?

Comment (weargoggles) [inline bin/compile]: I have shamelessly copied and pasted this from Jamie's known-working script here https://github.com/dabapps/heroku-buildpack-django-topography/blob/master/bin/compile#L3-L14

The colon is actually a builtin command which returns a success exit code. I'm not certain why that would be required here.

Comment (j4mie) [inline bin/compile]: I think I just copied and pasted it from somewhere else :)

**Comment by j4mie** [comment]
If we wanted to be really explicit, we could even have two env vars - `PROCFILE_PATH` and `ENVFILE_PATH`..

**Comment by weargoggles** [comment]
@j4mie I've been less explicit and more catfish specific. After all, the buildpack is called 'heroku-buildpack-catfish'!

**Comment by j4mie** [comment]
True, but that doesn't mean the individual bits of it should be explicit. If we don't do that, we'll have to cleverly handle all of the places that Catfish knows to look for environments. Which is totally fine - just have to pick one way or the other.

**Comment by weargoggles** [comment]
That's what I've done. catfish/ has a higher priority than etc/environments, should both exist.

**Comment by j4mie** [comment]
Excellent!

**Pull request merged**